stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -102,6 +102,24 @@ func.func @cholesky(%arg0: tensor<1x2x2xf32>) -> tensor<1x2x2xindex> {
 
 // -----
 
+// CHECK-LABEL: func @all_reduce_c7
```

Comment:
Just "all_reduce".

Reply:
This is the [guideline](https://github.com/openxla/stablehlo/blob/main/docs/reference_checklist.md) we've been following:

6. In [infer_stablehlo.mlir](https://github.com/openxla/stablehlo/blob/main/stablehlo/tests/infer_stablehlo.mlir):
i. Make sure all constraints related to shape inference tests are present in this file, following the same naming guidelines noted above.